src/DynamoDbModel.php

```diff
@@ -155,11 +155,57 @@ public function save(array $options = [])
         return $saved;
     }
 
+    /**
+     * Saves the model to DynamoDb asynchronously
+     * And returns a promise
```

Comment:
nit: `Saves the model to DynamoDb asynchronously and returns a promise` (don't capitalize `and`)